pkg/doc/verifiable/presentation.go

```diff
@@ -173,8 +173,8 @@ func (vp *Presentation) MarshalJSON() ([]byte, error) {
 
 // JWTClaims converts Verifiable Presentation into JWT Presentation claims, which can be than serialized
 // e.g. into JWS.
-func (vp *Presentation) JWTClaims(audience []string, minimizeVc bool) *JWTPresClaims {
-	return newJWTPresClaims(vp, audience, minimizeVc)
+func (vp *Presentation) JWTClaims(audience []string, minimizeVp bool) *JWTPresClaims {
```

Comment:
(small detail)
VP is an acronym right? So I think we usually would capitalize the "p" in "minimizeVp", kind of like how we capitalize "HTTP" in inbound.validateHTTPMethod.

Reply:
More specifically, I think acronyms generally need to have the same capitalization (so "vp *Presentation" is OK since "vp" is kind of like one word. Hope that makes sense)

Reply:
Good one! It should be `minimizeVP` of course. I'll fix that.

Reply:
Thanks! Looks good now